contrib/mypy/src/python/pants/contrib/mypy/tasks/mypy_task.py

```diff
@@ -36,6 +49,8 @@ def register_options(cls, register):
     register('--mypy-version', default='0.710', help='The version of mypy to use.')
     register('--config-file', default=None,
              help='Path mypy configuration file, relative to buildroot.')
+    register('--whitelisted-tag-name', default='type_checked',
```

Comment:
We'll want this to be an optional feature. You should be allowed to ignore this whitelist feature and just run `./pants lint.mypy ::`. For example, if the entire codebase has 100% type coverage, a whitelist wouldn't make sense for that user.

To do this, set the default to `None` and change the below logic to only filter if this whitelist option is provided.

Reply:
Also, maybe call this `--whitelist-tag-name`.